src/setup/devnet/index.js

```diff
@@ -1389,17 +1394,30 @@ export class Devnet {
         this.config.genesisAddresses = genesisAddresses
 
         // setup accounts from signer dump data (based on number of validators)
-        this.config.accounts = this.signerDumpData
-          .slice(0, this.config.numOfBorValidators)
-          .map((s) => {
-            return getAccountFromPrivateKey(s.priv_key)
-          })
+        //this.config.accounts = this.signerDumpData
+        //  .slice(0, this.config.numOfBorValidators)
+        //  .map((s) => {
+        //    //return getAccountFromPrivateKey(s.priv_key)
+        //      const account = getAccountFromPrivateKey(s.priv_key);
+        //      return { ...account, pub_key: s.pub_key };
+        //  })
+this.config.accounts = this.signerDumpData
+  .slice(0, this.config.numOfBorValidators)
+  .map((s) => {
+    const account = getAccountFromPrivateKey(s.priv_key);
```

Comment:
let's avoid semicolons. Get help from prettier/linter to auto solve all these issues.

Reply:
Also there are some redundant initializers and unused methods in index files. Let's take this refactoring opportunity to clean all of this up.